### DISCUSSION TITLE
Usage with GITHUB_TOKEN

### DISCUSSION THREAD

**Discussion by tonywok** [git-ecosystem/git-credential-manager#1359] [Q&A] [answered]
Hi there,

Looking at the source, it's unclear to me whether it's expected that GCM is designed to work with [`GITHUB_TOKEN`](https://docs.github.com/en/actions/security-guides/automatic-token-authentication).

Was hoping someone might be able to clarify.

Thanks :D

**Comment by ldennington** [comment, marked as answer]
It currently does not, but we are planning to add support for it. See [this issue](https://github.com/git-ecosystem/git-credential-manager/issues/1271) for details.

**Reply by tonywok** [reply]
Thanks for clarifying. I saw this issue, but was (hoping) it was more of an auto-configuration, rather than the feature itself. Look forward to it!
